packages/frameworks/src/frameworks.ts

```diff
@@ -2006,6 +2006,51 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Sanity',
+    slug: 'sanity',
+    demo: 'https://www.sanity.io/get-started',
+    logo: 'https://cdn.sanity.io/images/3do82whm/next/cd02ce807fbd324cbda82a77cd726c10ff6074d1-128x128.svg',
+    tagline:
+      'The structured content platform.',
+    description: 'A Sanity Studio',
+    website: 'https://www.sanity.io',
+    sort: 2,
+    envPrefix: 'SANITY_STUDIO_',
+    detectors: {
+      every: [
+        {
+          path: 'sanity.json',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`sanity install`',
```

Comment:
`sanity` CLI won't be present in the build container at the time the install command is executed, so let's leave this like the others:

```suggestion
        placeholder: '`yarn install` or `npm install`',
```